final_task/rss_reader/articles_handler.py

```diff
@@ -0,0 +1,201 @@
+import dataclasses
+import datetime
+import html
+import json
+import logging
+import os
+from typing import List, Tuple
+
+import dateutil.parser
+from bs4 import BeautifulSoup
+import feedparser
+
+import custom_error
+import single_article
+
+directory_to_module = os.path.abspath(os.path.dirname(__file__))
+CACHE_FILE_NAME = directory_to_module + '\\cache.json'
```

Comment:
It is better to use `os.path.join` instead of simple concatenation, because Windows and Linux have different path separators.

Same for logging file